LICENSE.txt

```diff
@@ -301,5 +301,39 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
+
+
+***************
+
+
+
+The software includes the FuzzySharp library.  The MIT License set out below is provided for informational purposes only. It is not the license that governs any part of the software.
```

Comment:
Just curious, where is the guideline to add license for assembly outside

Reply:
FuzzySharp is [MIT licensed](https://github.com/JakeBayer/FuzzySharp/blob/master/LICENSE). It requires us to put the copyright and license notice in our product.